docs/index.md

```diff
@@ -44,84 +44,46 @@
 
-1. Run the hello world app:
+2. Run the file with:
 
-   ```bash
-   streamlit hello
-   ```
+   `streamlit run [filename]`
 
-1. That's it! In the next few seconds the sample app will open in a new tab in
-   your default browser.
+3. When you're ready, click 'Deploy' from the Streamlit menu and use Streamlit sharing to share your app with the world!
 
-## Get started
+Now that you're set up, let's dive into more of how Streamlit works and how to build great apps.
 
-The easiest way to learn how to use Streamlit is to actually try it out. Our
-[get started guide](getting_started.md) walks you through the basics
-of building a Streamlit app.
+## How to use our docs
 
-## Build your first app
+The docs are broken up into 5 sections that will help you get the most out of Streamlit.
 
-```eval_rst
-.. raw:: html
-
-  <div class='embed-container'><iframe
-    width="560"
-    height="315"
-    src="https://www.youtube.com/embed/VtrFjkSGgKM"
-    style="margin: 0 0 2rem 0;"
-    frameborder="0"
-    allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
-    allowfullscreen>
-  </iframe></div>
-```
+**Tutorials**: include our [Get Started](getting_started.md) guide and a few step-by-step examples to building different types of apps in Streamlit.
+
+**Topic guides**: give you background on how different parts of Streamlit work. Make sure to check out the sections on [Creating an app](main_concepts.md) and [Deploying an app](deploy_streamlit_app.md), and for you advanced users who want to level up your apps, be sure to read up on [Caching](caching.md) and [Components](develop_streamlit_components.md).
+
+**Cookbook**: provides short code snippets that you can copy in for specific use cases.
+
+**Reference guides**: are the bread and butter of how our [APIs](api.md) and [configuration files](streamlit_configuration.md) work and will give you short, actionable explanations of specific functions and features.
 
-[Create an app](tutorial/create_a_data_explorer_app.md) to explore a dataset of
-Uber ride pickups in New York City. You'll learn about caching, drawing charts,
-plotting data on a map, and how to use interactive widgets.
+**Support**: gives you more options for when you're stuck or want to talk about an idea. Check out our discussion forum as well as a number of [troubleshooting guides](/troubleshooting/index.md).
 
-## Join the community
+## **Join the community**
 
-The quickest way to get help is to reach out on our [community
-forum](https://discuss.streamlit.io/). We love to hear our users' questions,
-ideas, and bugs — please share!
+Streamlit is more than just an app creation tool, it's also a community of creators that share their apps and ideas and help each other make their work better. Please come join us on the [community forum](https://discuss.streamlit.io/). We love to hear your questions, ideas, and help you work through your bugs — stop by today!
```

Comment:
nit: "more than just a way to make apps"?

Reply:
Agreed